app/drivers/zephyr/dts/bindings/led/is31fl3733/issi,is31fl3733.yaml

```
@@ -0,0 +1,9 @@
+description: |
+  Driver for the is31fl3733 LED driver
+
+compatible: "issi,is31fl3733"
+
+properties:
+  label:
+    required: true
+    type: string
```

This should be replaced with:

```suggestion
include: i2c-device.yaml
```

Which adds the `label` and `reg` property as expected, and sets `on-bus: i2c` as well.

---

prefect!